api/server/config.go

```diff
@@ -8,9 +8,10 @@ import (
 type Config struct {
 	Port          string
 	EtcdEndpoints []string
+	esURL         string
```

Comment:
I think it's ok to expose this as a public field, and the field name might as well be a bit more descriptive. Maybe `ElasticsearchURL`?